config.yml

```diff
@@ -0,0 +1,323 @@
+      - name: name
+      - name: type
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::ClassVariable
+    fields:
+      - name: name
+      - name: type
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::Extend
+    fields:
+      - name: name
+      - name: args
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::Include
+    fields:
+      - name: name
+      - name: args
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::InstanceVariable
+    fields:
+      - name: name
+      - name: type
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::MethodDefinition
+    fields:
+      - name: name
+      - name: kind
+      - name: overloads
+      - name: annotations
+      - name: location
+      - name: comment
+      - name: overloading
+      - name: visibility
+  - name: RBS::AST::Members::MethodDefinition::Overload
+    fields:
+      - name: annotations
+      - name: method_type
+  - name: RBS::AST::Members::Prepend
+    fields:
+      - name: name
+      - name: args
+      - name: annotations
+      - name: location
+      - name: comment
+  - name: RBS::AST::Members::Private
+    fields:
+      - name: location
+  - name: RBS::AST::Members::Public
+    fields:
+      - name: location
+  - name: RBS::AST::TypeParam
+    fields:
+      - name: name
+      - name: variance
+      - name: upper_bound
+      - name: default_type
+      - name: location
+  - name: RBS::MethodType
+    fields:
+      - name: type_params
+      - name: type
+      - name: block
+      - name: location
+  - name: RBS::Namespace
+    fields:
+      - name: path
+      - name: absolute
+  - name: RBS::TypeName
+    fields:
+      - name: namespace
+      - name: name
+  - name: RBS::Types::Alias
+    fields:
+      - name: name
+        c_name: typename # Temporary name to match existing C code. Will be removed in next PR.
+      - name: args
+      - name: location
+  - name: RBS::Types::Bases::Any
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Bool
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Bottom
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Class
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Instance
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Nil
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Self
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Top
+    fields:
+      - name: location
+  - name: RBS::Types::Bases::Void
+    fields:
+      - name: location
+  - name: RBS::Types::Block
+    fields:
+      - name: type
+      - name: required
+      - name: self_type
+  - name: RBS::Types::ClassInstance
+    fields:
+      - name: name
+        c_name: typename  # Temporary name to match existing C code. Will be removed in next PR.
```

Comment:
The identifiers used in C don't always match those used in the Ruby API. Where necessary, we used this `c_name:` field to preserve the C code as it was.

Reply:
On the actual PR against `ruby/rbs`, let's move this comment to line 212 👍